contracts/KyberPoolMaster.sol

```diff
@@ -53,43 +59,57 @@ contract KyberPoolMaster is Ownable {
     IERC20 public kncToken;
     IExtendedKyberDao public kyberDao;
     IKyberStaking public kyberStaking;
-    IExtendedKyberFeeHandler public kyberFeeHandler;
+
+    address[] public feeHandlersList;
+    mapping(address => IERC20) public rewardTokenByFeeHandle;
```

Comment:
Let's rename the variable for consistency. It will affect other lines.
```suggestion
    mapping(address => IERC20) public rewardTokenByFeeHandler;
```